src/extensions/common-api/index.ts

```diff
@@ -34,4 +35,5 @@ export {
   Store,
   Types,
   Util,
+  UserPreferences,
```

Comment:
I think `App.Preferences` would make more sense (and we wouldn't need to create another top-level namespace). Wdyt?

Reply:
sure